landlab/components/soil_grading/soil_grading.py

```diff
@@ -439,14 +439,14 @@ def create_dist(
             )
 
     def _generate_normal_distribution(
-        self, median_size=None, std=None, total_soil_weight=None
+        self, median_size = None, std = None, total_soil_weight = None
```

Comment:
Suggest docstring to explain what the function does / how it works.

Reply:
Added